tuxemon/core/player.py

```diff
@@ -47,5 +47,7 @@ def __init__(self, npc_slug):
         super(Player, self).__init__(npc_slug)
         self.isplayer = True
 
-        # Game variables for use with events
-        self.game_variables = {}
+        # Game variables for use with events:
+        self.game_variables['battle_history'] = {} # Maps the slugs of NPCs the Player has battled to the outcome of their battle

```

Comment:
The comments all the way on the right is a bit ugly,  I'd prefer this.
 \# Maps the slugs of NPCs the Player has battled to the outcome of their battle
 \# any NPC slug not present in the dictionary implies the Player has never 
 \# battled them before
 self.game_variables['battle_history'] = {}

Reply:
it's also technically a pep8 style violation, pushing the line length past 120 characters